snuba/admin/clickhouse/system_queries.py

```diff
@@ -178,9 +179,18 @@ def run_system_query_on_host_with_sql(
     clickhouse_host: str, clickhouse_port: int, storage_name: str, system_query_sql: str
 ) -> ClickhouseResult:
     validate_system_query(system_query_sql)
-    return _run_sql_query_on_host(
-        clickhouse_host, clickhouse_port, storage_name, system_query_sql
-    )
+    try:
+        return _run_sql_query_on_host(
+            clickhouse_host, clickhouse_port, storage_name, system_query_sql
+        )
+    except ClickhouseError as exc:
+        TABLE_DOES_NOT_EXIST = 60
+        MISSING_COLUMN = 47
```

Comment:
Can we use the actual error codes definition from clickhouse driver instead of redefining them?

Reply:
yes, fixed